api/types/server.go

```diff
@@ -335,7 +334,18 @@ func (s *ServerV2) CheckAndSetDefaults() error {
 
 // DeepCopy creates a clone of this server value
 func (s *ServerV2) DeepCopy() Server {
```

Comment:
A few other things to copy manually:
- `s.Spec.CmdLabels`
- `s.Spec.Apps`
- `s.Spec.KubernetesClusters`